packages/react-dom/package.json

```diff
@@ -48,8 +48,9 @@
     },
     "./profiling": "./profiling.js",
     "./test-utils": "./test-utils.js",
-    "./package.json": "./package.json",
-    "./": "./"
+    "./unstable_testing": "./unstable_testing.js",
+    "./umd/*": "./umd/*",
```

Comment:
I kept the UMD bundles reachable for now until we make a final decision on when to remove them